dcc-repository-azure/.gitignore

```diff
@@ -0,0 +1,2 @@
+/.apt_generated
+/target/
```

Comment:
turns out all the repositories have the same settings files. I didn't generate these (not using eclipse), they just got added from when I copied the AWS folder. I'll remove the settings folder for Azure since they arent necessary, but I won't remove the settings from the existing folders.